README.md

```diff
@@ -1,2 +1,79 @@
 # rhdh-plugin-export-utils
 Utilities for exporting backstage plugins as dynamic plugins for installation in Red Hat Developer Hub
+
+## Actions
+
+### export-dynamic
+
+Exports plugins as dynamic plugin archives. Supports per-plugin source overlays and should be run **after** `override-sources` if patch modifications are needed.
+
+**Usage:**
+```yaml
+- name: Export Dynamic Plugins
+  uses: ./export-dynamic
+  with:
+    plugins-root: plugins
+    plugins-file: ${{ github.workspace }}/plugins-list.yaml
+    destination: ${{ github.workspace }}/archives
+    source-overlay-folder-name: overlay  # optional, defaults to "overlay"
+```
+
+**Key Features:**
+- Exports plugins as dynamic plugin packages
+- Supports per-plugin source overlays with flexible directory structure:
```

Comment:
Should we mention that here, knowing that it will be removed from this action soon after ?